backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/init/CassandraTableManager.java

```diff
@@ -19,16 +19,20 @@
 
```

Comment:
this changeset must be split in several pieces, there are very different things here, no so hard to rework